gazebo/igvc_description/urdf/models/grass_plane/model.config

```diff
@@ -4,7 +4,6 @@
   <name>Grass Plane</name>
   <version>1.0</version>
   <sdf version="1.2">model-1_2.sdf</sdf>
-  <sdf version="1.3">model-1_3.sdf</sdf>
```

Comment:
I do not think this is a breaking change since we should all be above this version of sdf but I was just wondering if there was a reason this was deleted.

Reply:
We have no model-1_3.sdf file. I think we talked about this before, but it wasn't doing anything anyway

Reply:
yep yep, that is my bad.